src/lib/serviceRoutines/postUpdateContext.cpp

```diff
@@ -729,6 +721,10 @@ std::string postUpdateContext
       // If some CER (but not all) fails, then it is a partial update
       parseDataP->upcrs.res.oe.fill(SccContextElementNotFound, "Attributes that were not updated: { " + failing + " }", "PartialUpdate");
     }
+    else {  // fails == 0
+      // No fail in this case, so invalidate any posible OrionError filled by mongoBackend on the mongoUpdateContext ste
+      parseDataP->upcrs.res.oe.fill(SccNone, "");
```

Comment:
`SccNone` is a little dangerous (can give a `500` response if not modified).
Are we sure that the `SccNone` is always overwritten?
  
But, I imagine you have run the func tests ...

Reply:
SccNone is used by the NGSIv2 caller to check if an error has to be rendered. For instance, in postBatchUpdate(), just after calling postUpdateContext(), we have:

```
  // Check potential error
  if (parseDataP->upcrs.res.oe.code != SccNone )
  {
    TIMED_RENDER(answer = parseDataP->upcrs.res.oe.toJson());
    ciP->httpStatusCode = parseDataP->upcrs.res.oe.code;
  }
  else
  {
    ciP->httpStatusCode = SccNoContent;
  }
```

In addition, func test have passed, which validates the solution.

Reply:
Yeah, I imagined you had the tests run already.

Nothing to do with this PR but ...
What if the oe.code is 200 ?
A bit risky that line checking only SccNone and not SccOk ...

NTC